### PR TITLE
Returns URI fragments

### PR DESCRIPTION
**Pull request by quinnjn**
Adding parity for the [iOS PR](https://github.com/braintree/popup-bridge-ios/pull/13)

Question: Do we want the key to be `fragment`, or `hash`? Any other suggestions?

### CODE REVIEW

**Comment by skunkworks** [comment]
I chose `hash` to match the JS URL spec, although that may be a non-goal considering that we return `queryItems` and not `query`.

File: PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java [hunk 1]
```diff
@@ -139,6 +144,7 @@ public void onBrowserSwitchResult(int requestCode, BrowserSwitchResult result, U
             try {
                 json.put("path", returnUri.getPath());
                 json.put("queryItems", queryItems);
+                json.put("fragment", fragment);
```
Comment (skunkworks) [inline PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java]: iOS is omitting the `hash` aka `fragment` key/value pair if it's not defined. Any thoughts on which way we should go?

Comment (crookedneighbor) [inline PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java]: looks like ios is calling this `hash`, right? https://github.com/braintree/popup-bridge-ios/commit/cd45320#diff-7d6a776faf444a83619c6174c700ef2bR138

Comment (quinnjn) [reply PopupBridge/src/main/java/com/braintreepayments/popupbridge/PopupBridge.java]: Omitting it sounds good to me. I'll make the change.

**Pull request merged**